test/k8sT/PoliciesNightly.go

```diff
@@ -53,17 +54,13 @@ var _ = Describe("NightlyPolicies", func() {
 		kubectl.ValidateNoErrorsOnLogs(CurrentGinkgoTestDescription().Duration)
 	})
 
-	AfterEach(func() {
-		ExpectAllPodsTerminated(kubectl)
-	})
-
 	AfterAll(func() {
 		// Delete all pods created
 		kubectl.Exec(fmt.Sprintf(
 			"%s delete pods,svc,cnp -n %s -l test=policygen",
 			helpers.KubectlCmd, helpers.DefaultNamespace))
-
-		ExpectAllPodsTerminated(kubectl)
+		err := kubectl.WaitCleanAllTerminatingPods(timeout)
```

Comment:
Can we use `helpers.HelperTimeout` here?

Reply:
Too small value.